webapp/components/_new/features/SearchResults/SearchResults.story.js

```diff
@@ -0,0 +1,148 @@
+import { storiesOf } from '@storybook/vue'
+import { withA11y } from '@storybook/addon-a11y'
+import SearchResults from './SearchResults'
+import TabNavigation from '~/components/_new/generic/TabNavigation/TabNavigation'
+import PostTeaser from '~/components/PostTeaser/PostTeaser'
+import UserTeaser from '~/components/UserTeaser/UserTeaser'
+import helpers from '~/storybook/helpers'
+import faker from 'faker'
+import { post } from '~/components/PostTeaser/PostTeaser.story.js'
+import { user } from '~/components/UserTeaser/UserTeaser.story.js'
+
+helpers.init()
+
+const postMock = (fields) => {
+  return {
+    ...post,
+    id: faker.random.uuid(),
+    createdAt: faker.date.past(),
+    updatedAt: faker.date.recent(),
+    deleted: false,
+    disabled: false,
+    typename: 'Post',
+    ...fields,
+  }
+}
+
+const userMock = (fields) => {
+  return {
+    ...user,
+    id: faker.random.uuid(),
+    createdAt: faker.date.past(),
+    updatedAt: faker.date.recent(),
+    deleted: false,
+    disabled: false,
+    typename: 'User',
+    ...fields,
+  }
+}
+
+const posts = [
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-05-13T17:52:16Z" title="Wednesday, May 13th 2020, 7:52:16 pm +02:00">May 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,148 @@
+import { storiesOf } from '@storybook/vue'
+import { withA11y } from '@storybook/addon-a11y'
+import SearchResults from './SearchResults'
+import TabNavigation from '~/components/_new/generic/TabNavigation/TabNavigation'
+import PostTeaser from '~/components/PostTeaser/PostTeaser'
+import UserTeaser from '~/components/UserTeaser/UserTeaser'
+import helpers from '~/storybook/helpers'
+import faker from 'faker'
+import { post } from '~/components/PostTeaser/PostTeaser.story.js'
+import { user } from '~/components/UserTeaser/UserTeaser.story.js'
+
+helpers.init()
+
+const postMock = (fields) => {
+  return {
+    ...post,
+    id: faker.random.uuid(),
+    createdAt: faker.date.past(),
+    updatedAt: faker.date.recent(),
+    deleted: false,
+    disabled: false,
+    typename: 'Post',
+    ...fields,
+  }
+}
+
+const userMock = (fields) => {
+  return {
+    ...user,
+    id: faker.random.uuid(),
+    createdAt: faker.date.past(),
+    updatedAt: faker.date.recent(),
+    deleted: false,
+    disabled: false,
+    typename: 'User',
+    ...fields,
+  }
+}
+
+const posts = [
```


Use the helpers in `~/storybook/helpers.js`?
`fakeUser(n)`
`fakePost(n)`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-27T11:45:59Z" title="Thursday, August 27th 2020, 1:45:59 pm +02:00">Aug 27, 2020</time>_

---


See @Mogge comment above ☝🏼

This seems to be still valid. Or?

@mogge said:

Use the helpers in `~/storybook/helpers.js`?
`fakeUser(n)`
`fakePost(n)`